pytissueoptics/rayscattering/photon.py

```diff
@@ -16,10 +17,10 @@ def __init__(self, position: Vector, direction: Vector):
         self._direction = direction
         self._weight = 1
 
-        self._material = None
+        self._environment: Environment = None
```

Comment:
If we hint `Environment` and `Logger`, should also hint `FresnelIntersection` and `IntersectionFinder`. But they yellow flag because initialised to None